Source/Charts/Renderers/PieChartRenderer.swift

```diff
@@ -24,15 +24,15 @@ open class PieChartRenderer: NSObject, DataRenderer
     public final var accessibleChartElements: [NSUIAccessibilityElement] = []
 
     public let animator: Animator
-
+    
```

Comment:
You've introduce a lot of whitespace. I'm guessing you don't have "Automatically trim trailing whitespace>Including whitespace-only lines" enabled in Xcode>Preferences>TextEditing